docker/Dockerfile_stack

```diff
@@ -245,6 +246,8 @@ RUN echo '#!/usr/bin/env bash' > /usr/local/bin/mpiexec && \
 
 # setup vim
 COPY .vimrc /root/.vimrc
+# FIXME: workaround for CMake (< 3.11) to detect Boost.Python (>= 1.67) with version suffix
+RUN ln -s ${BOOST_DIR}/lib/libboost_python27.so ${BOOST_DIR}/lib/libboost_python.so
```

Comment:
Note that an alternative to this would be to install the latest cmake (3.11.1)
I have not verified whether it would let us work with clang-tidy then (c.f. #379)

Reply:
Still needed the workaround with cmake-3.11
I am wondering if it is because ycm try to find boost with the component python instead of pythonX.Y